RevenueCatUI/CustomerCenter/ViewModels/BaseManageSubscriptionViewModel.swift

```diff
@@ -100,6 +100,7 @@ class BaseManageSubscriptionViewModel: ObservableObject {
         switch path.detail {
         case let .feedbackSurvey(feedbackSurvey):
             self.feedbackSurveyData = FeedbackSurveyData(
+                productIdentifier: purchaseInformation?.productIdentifier ?? "-",
```

Comment:
Instead of defaulting to `"-"`, would it make sense for `FeedbackSurveyData`'s `productIdentifier` to be `optional`? This way, in the changed lines below, we would treat the `nil` case as the ineligible case 
```swift
if promotionalOffer.eligible, let productIdentifier = feedbackSurveyData?.productIdentifier {
... // Would not enter here if `productIdentifier` were `nil`
}
```

and the same would apply in `FeedbackSurveyViewModel` around line 70
```swift
if let promotionalOffer = option.promotionalOffer,
           promotionalOffer.eligible,
           let productIdentifier = feedbackSurveyData.productIdentifier {
... // Would not enter here if `productIdentifier` were `nil`
}
```